kvdb/etcd/config.go

```diff
@@ -81,6 +82,7 @@ func (c *Config) CloneWithSingleWriter() *Config {
 		KeyFile:            c.KeyFile,
 		InsecureSkipVerify: c.InsecureSkipVerify,
 		CollectStats:       c.CollectStats,
+		MaxMsgSize:         c.MaxMsgSize,
```

Comment:
Implications of this fix?

Reply:
The `MaxMsgSize` wasn't properly copied and we do copy these configs when `lncfg.GetBackends()`. Basically all clones were using default message size.